### PR TITLE
CNF-14676:numaresources: build bundle images

### PR DESCRIPTION
**Pull request by Tal-or**
Build bundle images for NROP and promote it to QCI, then mirror it to Quay.

We'll need this as a prerequisite for adding an upgrade test and it is good the have the bundles handy in general.

### CODE REVIEW

File: ci-operator/jobs/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main-presubmits.yaml [hunk 1]
```diff
@@ -156,23 +156,23 @@ presubmits:
     - ^main$
     - ^main-
     cluster: build05
-    context: ci/prow/ci-index-numaresources-operator-bundle
+    context: ci/prow/ci-index
```
Comment (ffromani) [inline ci-operator/jobs/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main-presubmits.yaml:159]: why do need this rename?

Comment (ffromani) [reply ci-operator/jobs/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main-presubmits.yaml:159]: And the following as well

Comment (Tal-or) [reply ci-operator/jobs/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main-presubmits.yaml:159]: This one auto generated once I removed the `bundle.as`:
 https://github.com/openshift/release/pull/58597/commits/2de0aaea74185f69203b6e51de025cdd4de52bc9#diff-b719e9d20a9992f73ccfc477a317704367093899f38d6915881e68cdb5af322fL51

Comment (ffromani) [reply ci-operator/jobs/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main-presubmits.yaml:159]: ok, seems no big deal

**Review by ffromani — COMMENTED** [review]
/lgtm

**Pull request merged**